roles/entitlement_deploy/files/apply_template.py

```diff
@@ -1,19 +1,33 @@
-#! /usr/bin/python3
+#!/usr/bin/env python3
 
-import fileinput
```

Comment:
Changed it so it no longer reads from a file, just stdin - so we can stack several apply_template.py calls in pipes